app/presenters/report_presenter.rb

```diff
@@ -34,26 +34,18 @@ def due_defects_by_priority(priority:)
   end
 
   def overdue_defects_by_priority(priority:)
-    defects = defects_by_priority(priority: priority)
-    defects_completed_late = defects.where('target_completion_date < actual_completion_date')
-    defects_still_open_and_overdue = defects.open.where('target_completion_date < ?', Date.current)
-
-    (defects_completed_late + defects_still_open_and_overdue).uniq
+    (
+      defects_completed_late(priority: priority) +
+      defects_still_open_and_overdue(priority: priority) +
+      completed_defects_with_no_completion_date(priority: priority)
```

Comment:
Do we know that this behaviour will make sense to users? Will this cause them to go "it's still broken"?

Reply:
I believe so, I've tested it out locally and compared the numbers to the overall data dump and the numbers all seem to add up. I'd like to test with the team on staging.